### ISSUE TITLE
Bug: axe-core/cli validator fails on ARIA 1.2 required owned elements when aria-busy=true

### ISSUE THREAD

**Issue by design1online** [dequelabs/axe-core-npm#292]
ARIA 1.2 spec states: There may be times that required owned elements are missing, for example, while editing or while loading a data set. When a widget is missing required owned elements due to script execution or loading, authors MUST mark a containing element with aria-busy equal to true. For example, until a page is fully initialized and complete, an author could mark the document element as busy. [See the ARIA documentation.](https://www.w3.org/TR/wai-aria-1.2/#mustContain)

**Product:** axe-core/cli

**Expectation:** When a child with required owned elements is loading dynamic content it should pass the validation if aria-busy is equal to true

**Actual:** Currently it fails with `Fix any of the following:  Required ARIA children role not present: group, listitem`

**Motivation:** Needs to validate to ARIA 1.2 spec

**Sample Failing Validation:**
<pre>
&lt;div id="while-loading" aria-label="while-loading" role="list"&gt;
  &lt;div class="loader" aria-busy="true"&lt;Loading List...&lt;/div&gt;
&lt;/div&gt;
</pre>

**System Specs:**
<pre><code>
- axe-core: 4.2.3
- Node version: 14.17.0
- Platform:  MacOS Catalina 10.15.7
</code></pre>

**Comment by Zidious** [comment]
Thank you for raising this issue. 

I have opened it within axe-core see ref: https://github.com/dequelabs/axe-core/issues/3208
